pkgs/by-name/lo/lofreq/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  autoreconfHook,
+  htslib,
+  python3,
+  zlib,
+
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lofreq";
+  version = "2.1.5";
+
+  src = fetchFromGitHub {
+    owner = "CSB5";
+    repo = pname;
```

- https://github.com/NixOS/nixpkgs/issues/277994
```suggestion
    repo = "lofreq";
```